charts/plex-media-server/Chart.yaml

```diff
@@ -22,13 +22,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.0
+version: 0.7.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "1.41.3"
```

Comment:
@MarshallAsch does this app bump look good or should this match the docker tag (currently `4.141.0` if I'm not mistaken)

Reply:
I changed it to `1.41.3` which appears to be the current `latest` pms docker image version. If I understand correctly,  `4.141.0` seems like plex-web version.

Either way, please let me know what is expected and I can change it.

Reply:
Can confirm: https://hub.docker.com/layers/plexinc/pms-docker/1.41.3.9314-a0bfb8370/images/sha256-19746abbf989b6be7b5798149567ef4c498a6a848b47f53d0dd948078b32eb0a - thanks for clarifying.